stratio-docs/en/modules/ROOT/pages/operations.adoc

```diff
@@ -0,0 +1,297 @@
+
+Once the manifest has been created, the creation of the group simply consists of applying it to the cluster as follows:
+
+[source,bash]
+----
+kubectl apply -f xref:attachment$example-eks-md.yaml[example-eks-md.yaml]
+----
+
+To view the created objects:
+
+[source,bash]
+----
+kubectl -n cluster-example get md,eksct,awsmt
+----
+
+===== GCP
+
+For GCP, the following will be created: _MachineDeployment_, _GCPMachineTemplate_ and _KubeadmConfigTemplate_.
+
+In the same way, the manifest is applied to create the new workers group:
+
+[source,bash]
+----
+kubectl apply -f xref:attachment$example-gcp-md.yaml[example-gcp-md.yaml]
+----
+
+To see the created objects:
+
+[source,bash]
+----
+kubectl -n cluster-example get md,gcpmachinetemplate,kubeadmconfigtemplate
+----
+
+==== Vertical scaling
+
+Vertical scaling of a node group can be done in several ways, all of which will start by changing the instance type of the `<infra-controller>MachineTemplate` object.
+
+TIP: Although the official guidelines require creating a new `<infra-controller>MachineTemplate` and referencing it from the _MachineDeployment_, this option is not recommended. It prevents maintaining naming consistency between the objects that manage the node groups.
+
+The recommended method is based on 3 simple steps:
+
+. specify the new instance type in `<infra-controller>MachineTemplate` (_spec.template.spec.instanceType_). In some vendors, this object will have to be deleted and created anew.
+. Get the version of the new `<infra-controller>MachineTemplate` object (_metadata.resourceVersion_).
+. Edit the _MachineDeployment_ by updating the version obtained in the previous step (_spec.template.spec.infrastructureRef.resourceVersion_).
+
+An example of an EKS cluster would be as follows:
+
+[source,bash]
+----
+export MACHINE_TYPE="t3.medium"
+export MACHINE_DEPLOYMENT="stg-eks-xlarge-md-2"
+export NAMESPACE="cluster-stg-eks"
+
+$ k -n $NAMESPACE patch awsmt $MACHINE_DEPLOYMENT --type merge -p "{\"spec\": {\"template\": {\"spec\": {\"instanceType\": "$MACHINE_TYPE"}}}}"
+
+$ RESOURCE_VERSION=$(k -n $NAMESPACE get awsmt $MACHINE_DEPLOYMENT -o json | jq -r .metadata.resourceVersion)
+
+$ k -n $NAMESPACE patch md $MACHINE_DEPLOYMENT --type merge -p "{\"spec\": {\"template\": {\"spec\": {\"infrastructureRef\": {\"resourceVersion\": \"$RESOURCE_VERSION\"}}}}}"
+----
+
+=== Autoscaling
+
+image::autoescalado.png[]
+
+The _cluster-autoscaler_ is used for node autoscaling. It will detect pods pending execution due to a lack of resources and it will scale groups of nodes according to the deployment filters.
+
+This operation is performed in the API server. The controllers are in charge of creating the VMs in the cloud provider and adding them to the cluster as Kubernetes worker nodes.
+
+Since the autoscaling is based on the _cluster-autoscaler_, the minimum and maximum will be added in the workers node group as annotations:
```

Comment:
```suggestion
Since the autoscaling is based on the _cluster-autoscaler_, the minimum and maximum will be added in the worker's node group as annotations:
```